### PR TITLE
Cambio de Allan y Eduardo

### PR DESCRIPTION
**Pull request by allanletona9**

### CODE REVIEW

**Review by keyshardk — APPROVED** [review]

**Comment by keyshardk** [comment]
RECHAZADO

**Comment by imogollonh** [comment]
RECHAZADOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO

**Comment by allanletona9** [comment]
No entiendo este rechazo.

File: index.html [hunk 1]
```diff
@@ -8,5 +8,6 @@
 </head>
 <body>
     <p>hola mundo</p>
+    <p>Kenia: Pais</p>
```
Comment (asolaresm3) [inline index.html:11]: Alejandra Reyna

**Pull request closed without merging**